### ISSUE TITLE
Desired behaviour for empty line

### ISSUE THREAD

**Issue by SalimBensiali** [knrz/CSV.js#31]
Hi,

Currently, the parser reads empty lines and sets default values as follows: 

If the first column is a number then it gets a `0`, otherwise if it is a string then it gets `""`. For the other columns numbers get `NaN` and strings get `undefined`.

```
var data = '\
year,age,status,sex,population\r\n\
1850,20,0,1,1017281\r\n
\r\n\
1850,20,0,2,1003841\r\n\
';
new CSV(data, { header: true }).parse();
/*
Returns:
[
  { year: 1850, age: 20, status: 0, sex: 1, population: 1017281 },
  { year: 0, age: NaN, status: NaN, sex: NaN, population: NaN },
  { year: 1850, age: 20, status: 0, sex: 2, population: 1003841 }
]

var data2 = '\
name,age\r\n\
bob,20\r\n
\r\n\
Chris,20\r\n\
';
new CSV(data2, { header: true }).parse();
/*
Returns:
[
  { name: "bob", age: 20 },
  { name: "", age: NaN },
  { name: "Chris", age: 20 }
]

*/
```

My initial assumption/hope was for the empty lines to simply be ignored.

I would love to hear your thoughts on that.

Cheers,
Salim

**Comment by knrz** [comment]
Huh. I never actually ran into empty lines. 

Seems like a good edge-case to accommodate. Let me look into it.

**Comment by knrz** [comment]
Fixed with version 3.6.4 :+1: :smiley: :shipit:

**Comment by SalimBensiali** [comment]
Thanks :)
